api/testdata/complete-scenarios/test3.json

```diff
@@ -2767,9 +2767,9 @@
                   "Relinquished": {
                     "type": "datecontrol",
                     "props": {
-                      "month": "04",
-                      "day": "04",
-                      "year": "1998",
+                      "month": "",
```

Comment:
For this `Relinquished` field, it has both a relinquished date as well as the relinquished date marked as not applicable. This was probably a case of data improperly getting cleared out then `Not applicable` was clicked.